command.py

```diff
@@ -1,14 +1,18 @@
-import discord
+"""
+Implementing CalicoBot commands
+"""
 import datetime
-from discord.ext import commands
-from discord import app_commands
 import time
 import embeds
-import main
+import discord
+from discord.ext import commands
+from discord import app_commands
 
-bot=commands.Bot(command_prefix="t?",intents=discord.Intents.all())
+user_karma_list=list()
+def command_init(main_list):
+    user_karma_list = main_list
```

Comment:
I wasn't understanding that It can't assign outside variables.
```suggestion
    global user_karma_list# pylint: disable=global-statement
    user_karma_list = main_list
```